packages/ui/components/ui/tx/view/transaction.tsx

```diff
@@ -8,7 +8,9 @@ export const TransactionViewComponent = ({ txv }: { txv: TransactionView }) => {
   if (!txv.bodyView) throw new Error('transaction view missing body view');
   if (!txv.bodyView.transactionParameters?.fee?.amount) throw new Error('Missing fee amount');
 
-  const fee = joinLoHiAmount(txv.bodyView.transactionParameters.fee.amount).toString();
+  const fee = (
+    Number(joinLoHiAmount(txv.bodyView.transactionParameters.fee.amount)) / 1000000
+  ).toString();
```

Comment:
We should probably use the `exponent` value from the registry metadata instead of the constant `1000000` here

Reply:
Definitely — especially if the fee can be in other currencies than UM

Reply:
modified to use `getStakingTokenMetaData` to retrieve the exponent relevant to the relevant fee asset_id in the TxV